rest/client/rest.go

```diff
@@ -657,3 +657,56 @@ func (c *communicatorImpl) CreateVersionFromConfig(ctx context.Context, project,
 
 	return v, nil
 }
+
+func (c *communicatorImpl) GetCommitQueue(ctx context.Context, projectID string) (*model.APICommitQueue, error) {
+	info := requestInfo{
+		method:  get,
+		version: apiVersion2,
+		path:    fmt.Sprintf("/projects/%s/commit_queue", projectID),
+	}
+
+	resp, err := c.request(ctx, info, "")
+	if err != nil {
+		return nil, errors.Wrap(err, "problem fetching commit queue list")
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		errMsg := gimlet.ErrorResponse{}
+
+		if err = util.ReadJSONInto(resp.Body, &errMsg); err != nil {
+			return nil, errors.Wrap(err, "problem fetching commit queue list and parsing error message")
+		}
+		return nil, errors.Wrap(errMsg, "problem fetching commit queue list")
```

Comment:
Don't wrap. (I'm sympathetic to wrapping the error from `ReadJSONInto`, since that could give a clue about why JSON unmarshalling failed, which is why I did not comment on 676 or 684, and also the error from `request`.)

Reply:
done.